libs/designer/src/lib/core/utils/outputs.ts

```diff
@@ -200,7 +200,12 @@ export const getUpdatedManifestForSplitOn = (manifest: OperationManifest, splitO
     }
 
     const manifestItems: OpenAPIV2.SchemaObject | undefined = clone(manifestSection.items);
-    const updatedManifestItems = isAliasPathParsingEnabled ? getUpdatedAliasInItemProperties(manifestItems) : manifestItems;
+
+    let updatedManifestItems = manifestItems;
+    if (isAliasPathParsingEnabled && updatedManifestItems) {
```

Comment:
I think you should be checking isAliasPathParsingEnabled && manifestItems && OperationManifestService().getUpdatedAliasInOutputIt....) { 
.... 
}

Reply:
Does that mean the manifest service function can replace the existing function?

Reply:
Does logic apps not use the aliasing logic at all? Is it safe to replace with a host service plug in?